src/App.js

```diff
@@ -110,8 +112,9 @@ class App extends Component {
       renderFooterPagesList.includes(pathname) ? (
         <Footer />
       ) : null;
-    const renderCookiePolicy = showCookiePolicy ? <CookiePolicy /> : null;
-    const renderDialog = isModalOpen ? <DialogSection /> : null;
+    const renderCookiePolicy =
+      showCookiePolicy === true ? <CookiePolicy /> : null;
+    const renderDialog = isModalOpen || !visited ? <DialogSection /> : null;
```

Comment:
Why have you used `|| !visited`? Won't it open DialogSection if `visited` is false? If you intend to open the Dialog, then where is the type of Dialog defined?

Reply:
The video tour dialog will not open if we don't use this condition, it opens when visited is set to false in `DialogSection`.